PDG/Fact fact
string agentName
int32 agentId
---
string answer